libsolidity/formal/SSAVariable.h

```diff
@@ -46,7 +46,7 @@ class SSAVariable
 	unsigned m_currentIndex;
 	/// The next free index is a shared pointer because we want
 	/// the copy and the copied to share it.
```

Comment:
This comment sounds like you did expect ``SSAVariable`` to be copiable at some point, but if this change works I guess you never actually copy it? So either there's something wrong here or you should change the comment.

Reply:
I was just thinking about that. I think this changed. It used to be the case that the SymbolicVariables were copied before/after branching, but now only the indices are.
I'll adjust the comment.